docs/config/experimental_world.md

```diff
@@ -90,7 +91,7 @@ Customize the message players receive when they fall below the minimum Y positio
 
 :::code-group
 ```toml [server.toml] {2}
-[world.experimental]
+[world.boundaries]
 min_y_message = "<red>You have reached the bottom of the world.</red>"
```

Comment:
The settings here should be renamed to match the new names, min_y and teleport_message